keylime/ima/ima.py

```diff
@@ -18,26 +18,20 @@
 logger = keylime_logging.init_logging("ima")
 
 
-# The version of the allowlist format that is supported by this keylime release
-ALLOWLIST_CURRENT_VERSION = 5
-
-# A correctly-formatted empty allowlist.
-#
-# IMA allowlists of versions older than 5 will not have the "log_hash_alg"
-# parameter. Hard-coding it to "sha1" is perfectly fine, and the fact one
-# specifies a different algorithm on the kernel command line (e.g., ima_hash=sha256)
```

Comment:
I think this comment should be still here, because otherwise the hardcoded value causes again some confusion.

Reply:
I've added the comment back in my latest revision.